array-manipulation-3/src/intersection.js

```diff
@@ -5,12 +5,16 @@ function intersection(first, second) {
     for (var x = 0; x < second.length; x++) {
       if (first[i] === second[x]) {
         output.push(first[i]);
+        continue;
```

Comment:
This won't fix the problem I mentioned. You need to check if that value is already in `output` before pushing it, so that you don't add duplicates.